org.desmume.DeSmuME.json

```diff
@@ -1,14 +1,16 @@
 {
     "app-id": "org.desmume.DeSmuME",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "20.08",
+    "runtime-version": "21.08",
     "sdk": "org.freedesktop.Sdk",
     "command": "desmume",
     "finish-args": [
         "--socket=wayland",
         "--socket=fallback-x11",
         "--socket=pulseaudio",
-        "--device=dri"
+        "--device=dri",
+        "--share=ipc",
+        "--device=all"
```

Comment:
```suggestion
        "--share=ipc",
        "--device=all"
```

all includes dri

Reply:
I'll assume this is really necessary (game controllers?)

Reply:
Yep, I've explained each new one in dedicated commits.

Reply:
Right

Reply:
Thanks, done. I've put it in the `device=all` commit